python/paddle/vision/__init__.py

```diff
@@ -12,21 +12,74 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from . import models
 from .models import *
 
-from . import transforms
 from .transforms import *
 
-from . import datasets
 from .datasets import *
 
-from . import image
 from .image import *
 
-from . import ops
 
-__all__ = models.__all__ \
-        + transforms.__all__ \
-        + datasets.__all__ \
-        + image.__all__
+__all__ = [     #noqa
```

Comment:
maybe places the models at `paddle.vision.models` level

Reply:
done